include/test/miral/test_display_server.h

```diff
@@ -63,6 +63,16 @@ struct TestDisplayServer : private TestRuntimeEnvironment
     /// \note call before start_server()
     void add_stop_callback(std::function<void()> const& stop_callback);
 
+    /// Add signal handler to the server's main loop.
+    /// \remark Since MirAL 3.7
```

Comment:
You should also bump `MIRAL_VERSION_MINOR` in `CMakeLists.txt` - that will allow downstream code to test for the version